sample/build.gradle

```
@@ -55,12 +53,12 @@ dependencies {
 
     // Compose
     implementation "androidx.activity:activity-compose:$composeActivity"
-    implementation "androidx.compose.ui:ui:1.7.0-alpha08"
+    implementation "androidx.compose.ui:ui:1.7.2"
     implementation "androidx.compose.ui:ui-tooling-preview:$compose"
 
     // Material
     implementation "androidx.compose.material:material:$material"
```

Upgrading to material 1.7.2 requires adding a separate dependency for icon resources.

```suggestion
    implementation "androidx.compose.material:material:$material"
    implementation "androidx.compose.material:material-icons-core-android:$material"
```

---

added with compose bom